docs/docs/attestation/sbom.md

```diff
@@ -32,7 +32,7 @@ $ cosign attest --key /path/to/cosign.key --type spdx --predicate sbom.spdx.json
 
 # cyclonedx
 $ trivy image --format cyclonedx -o sbom.cdx.json <IMAGE>
-$ cosign attest --key /path/to/cosign.key --type https://cyclonedx.org/schema --predicate sbom.cdx.json <IMAGE>
+$ cosign attest --key /path/to/cosign.key --type cyclonedx --predicate sbom.cdx.json <IMAGE>
```

Comment:
We probably should put a note about cosign version as `cyclonedx` was added in v1.10.0.

Reply:
I've added a description about cosign version.